### PR TITLE
Create JSON Transform mediator UI in management console

### PR DESCRIPTION
**Pull request by arunans23**
## Purpose
JSON Transform mediator was recently introduced (https://github.com/wso2/wso2-synapse/pull/1410) and this PR brings the UI support for it in the management console.

Related to https://github.com/wso2/product-ei/issues/4377

### CODE REVIEW

File: components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.jsontransform.ui/src/main/resources/web/jsontransform-mediator/update-mediator.jsp [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1" pageEncoding="ISO-8859-1" %>
+<%--
+  ~  Copyright (c) 2008, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (GDLMadushanka) [inline components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.jsontransform.ui/src/main/resources/web/jsontransform-mediator/update-mediator.jsp]: 2019

File: components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.jsontransform.ui/src/main/resources/web/jsontransform-mediator/update-mediator.jsp [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    MediatorProperty meditorProp = null;
+    if (!(mediator instanceof JSONTransformMediator)) {
+        // todo : proper error handling
+        throw new RuntimeException("Unable to edit the mediator");
+    }
+    JSONTransformMediator jsonTransformMed = (JSONTransformMediator) mediator;
+    jsonTransformMed.getProperties().clear(); // to avoid duplicates
+    
+    String propertyCountParameter = request.getParameter("propertyCount");
+    if (propertyCountParameter != null && !"".equals(propertyCountParameter)) {
```
Comment (GDLMadushanka) [inline components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.jsontransform.ui/src/main/resources/web/jsontransform-mediator/update-mediator.jsp]: shall we use isEmpty(0

File: components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.jsontransform.ui/src/main/resources/web/jsontransform-mediator/update-mediator.jsp (hunk 1 above)
Comment (arunans23) [reply components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.jsontransform.ui/src/main/resources/web/jsontransform-mediator/update-mediator.jsp]: Fixed

File: components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.jsontransform.ui/src/main/resources/web/jsontransform-mediator/update-mediator.jsp (hunk 2 above)
Comment (arunans23) [reply components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.jsontransform.ui/src/main/resources/web/jsontransform-mediator/update-mediator.jsp]: Fixed

File: components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.jsontransform.ui/src/main/resources/web/jsontransform-mediator/edit-mediator.jsp [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+        <table class="normal" width="100%">
+            <tr>
+                <td>
+                    <h2><fmt:message key="mediator.jsontransform.header"/></h2>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <table class="normal">
+                        <tr id="mediator.xslt.key.static">
```
[the remark this answers is not in the document]
Comment (arunans23) [reply components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.jsontransform.ui/src/main/resources/web/jsontransform-mediator/edit-mediator.jsp]: Fixed

**Pull request merged**